endoreg_db/models/medical/patient/patient_finding.py

```
@@ -26,116 +39,263 @@ class Meta:
-            self, location_classification
-        ):
-        """
-        Sets a random location for this finding based on the location classification.
-        """
+        return self.interventions.filter(is_active=True).select_related('intervention')
+
+    # Verbesserte Add-Methoden mit Validierung
+    def add_location(self, location_classification_id, location_choice_id, user=None):
+        """Adds a validated location choice to this patient finding."""
         from .patient_finding_location import PatientFindingLocation
+        from ..finding.location import FindingLocationClassification
```

Pylint (3.3.7)

[error] No name 'location' in module 'endoreg_db.models.medical.finding'

(E0611)